src/content/api/_endpoints/merchant-configs-create.js

```diff
@@ -13,7 +13,7 @@ export default {
           walletId: '1234c486308f3f0a23f0f92b'
         },
         {
-          type: 'epay.nzd.main',
+          type: 'epay.main',
```

Comment:
Looks like `epay.main` asset type requires `externalMerchantId`, not `terminalId`, based on the code.

Reply:
You are right good catch